examples/asr/conf/ssl/nest/multi_layer_feat/nest_ecapa_tdnn_small.yaml

```diff
@@ -0,0 +1,185 @@
+# This is an example config that uses a NEST model as feature extractors, by using multi-layer feature aggregation.
+# The major modification is to replace `model.preprocessor` with the one in this config. 
+
+name: "NEST_MFA_Tune_ECAPA_TDNN"
+
+model:
+  sample_rate: 16000
+
+  train_ds:
+    manifest_filepath: ???
+    sample_rate: ${model.sample_rate}
+    labels: null
+    batch_size: 64
+    num_workers: 8
+    shuffle: True
+    augmentor:
+      noise:
+        manifest_path: ???
+        prob: 0.5
+        min_snr_db: 0
+        max_snr_db: 15
+
+      speed:
+        prob: 0.5
+        sr: ${model.sample_rate}
+        resample_type: 'kaiser_fast'
+        min_speed_rate: 0.95
+        max_speed_rate: 1.05
+
+      impulse:
+        prob: 0.5
+        manifest_path: ???
+
+  validation_ds:
+    manifest_filepath: ???
+    sample_rate: ${model.sample_rate}
+    labels: null
+    batch_size: 128
+    num_workers: 8
+    shuffle: False
+
+  # Overwrite the original 'preprocessor' and replace it with multi-layer feature aggregation from ConformerEncoder
+  preprocessor:
+    _target_: nemo.collections.asr.modules.ssl_modules.multi_layer_feat.ConformerMultiLayerFeaturePreprocessor
+    layer_idx_list: [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16]
+    freeze_encoder: True
+    aggregator:
+      _target_: nemo.collections.asr.modules.ssl_modules.multi_layer_feat.Aggregator
+      mode: "weighted_sum"
+      weights: null
+      layer_idx_list: ${model.preprocessor.layer_idx_list}
+    
+    # the actual preprocessor to use
+    preprocessor:
+      _target_: nemo.collections.asr.modules.AudioToMelSpectrogramPreprocessor
+      normalize: "per_feature"
+      window_size: 0.025
+      sample_rate: ${model.sample_rate}
+      window_stride: 0.01
+      window: "hann"
+      features: 80
+      n_fft: 512
+      frame_splicing: 1
+      dither: 0.00001
+      stft_conv: false
+
+    spec_augment:
+      _target_: nemo.collections.asr.modules.SpectrogramAugmentation
+      freq_masks: 3
+      freq_width: 4
+      time_masks: 5
+      time_width: 0.05
+    
+    # this has to match with the ConformerEncoder config in NEST pretrain config
+    encoder:
```

Comment:
that would make it difficult for general nemo asr models to use, since all are using `model.preprocessor`. the reason for keeping the same name is to let arbitrary nemo asr models to use the MFA preprocessor